src/game/UICmdEdit.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include "GameDef.h"
+#include "N3Base/N3UIBase.h"
+#include "N3Base/N3UIString.h"
+#include "N3Base/N3UIEdit.h"
+
+//////////////////////////////////////////////////////////////////////
+
+class CUICmdEditDlg : public CN3UIBase {
+  public:
+    CN3UIString * m_pText_Title;
+    CN3UIButton * m_pBtn_Ok;
+    CN3UIButton * m_pBtn_Cancel;
+    CN3UIEdit *   m_pEdit_Box;
+    std::string   m_szArg1;
+
+  public:
+    void SetVisible(bool bVisible);
```

Comment:
This method is virtual but doesn't indicate it as such. Ideally it should be using `override` (rather than refining it as `virtual` to enforce the overridden virtual method (that way it can error if it's defined incorrectly).
i.e.
```cpp
    void SetVisible(bool bVisible) override;
```